src/pages/task/actionButtons/RelevantCommits/index.tsx

```diff
@@ -37,101 +32,18 @@ export const RelevantCommits: React.FC<RelevantCommitsProps> = ({ taskId }) => {
     variables: { taskId },
   });
 
-  const {
-    baseTask,
-    buildVariant,
-    displayName,
-    projectIdentifier,
-    status,
-    versionMetadata,
-  } = taskData?.task ?? {};
-  const { order: skipOrderNumber } = versionMetadata?.baseVersion ?? {};
+  const { baseTask, versionMetadata } = taskData?.task ?? {};
```

Comment:
I just realized that there doesn't seem to be a reason to perform the `BaseVersionAndTaskQuery` at all; it looks like the parent component of this component (`ActionButtons.tsx`) can pass these values down as props 🤔